python/samples/getting_started/chat_store/third_party_chat_store_cosmosDB.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+"""Cosmos DB Chat Message Store Example
+
+Demonstrates how to store and retrieve chat history using Azure Cosmos DB
+as an external message store for the Microsoft Agent Framework.
+
+Scenarios:
+  1) Persist chat messages in Cosmos DB with thread-based partitioning.
+  2) Retrieve messages in chronological order for conversation continuity.
+
+Requirements:
+  - Azure Cosmos DB (Core SQL API) with an existing database and container.
+  - Container partition key must be /thread_id.
+  - Environment variables:
+      COSMOS_DB_ENDPOINT,
+      AZURE_OPENAI_ENDPOINT, AZURE_OPENAI_API_KEY,
+      AZURE_OPENAI_API_VERSION, AZURE_OPENAI_CHAT_DEPLOYMENT_NAME
+  - Dependencies:
+      pip install azure-cosmos pydantic "agent-framework"
```

Comment:
Pydantic is already part of the AF Core Package

```suggestion
      pip install azure-cosmos "agent-framework"
```